tests_common/pytest_plugin.py

```diff
@@ -1606,6 +1607,18 @@ def _disable_redact(request: pytest.FixtureRequest, mocker):
     return
 
 
+@pytest.fixture(autouse=True)
+def _mock_plugins(request: pytest.FixtureRequest):
+    """Disable redacted text in tests, except specific."""
+    if mark := next(request.node.iter_markers("mock_plugin_manager"), None):
+        from tests_common.test_utils.mock_plugins import mock_plugin_manager
+
+        with mock_plugin_manager(**mark.kwargs):
+            yield
+            return
+    yield
```

Comment:
This is important to introduce because I was running into a scenario where the `mock_plugin_manager` as a wrapper was running before the dag was serialising, leading to some issues like the test plugins loading when I didnt wish for them too.